contracts/apps/ownership/OwnershipApp.sol

```diff
@@ -96,8 +124,8 @@ contract OwnershipApp is Application, Controller, Requestor {
   }
 
   function isHolder(address _holder) constant returns (bool) {
-    uint tokenCount = tokens.length;
-    for (uint i = 0; i < tokenCount; i++) {
+    uint tokenCount = getTokenCount();
```

Comment:
GIven that getTokenCount() has specific logic `tokens.length - 1`, i'd rather just have it once. 

Also i'm not sure how expensive internal functions are gas wise, researching it